### PR TITLE
Part 1 - core feature - allow pseudo comment exception #329

### PR DESCRIPTION
**Pull request by eugen-guenther-sap**

### CODE REVIEW

File: src/profiles/y_profile_admin_classes.prog.abap [hunk 1]
```diff
@@ -558,6 +556,8 @@ CLASS lcl_util IMPLEMENTATION.
                                             header_text = 'Apply on Productive Code'(050) ).
         checks_tree->set_field_header_text( fieldname   = 'APPLY_ON_TESTCODE'
                                             header_text = 'Apply on Testcode'(034) ).
+        checks_tree->set_field_header_text( fieldname   = 'IGNORE_PSEUDO_COMMENTS'
+                                            header_text = 'Allow Pseudo Comments' ).
```
Comment (lucasborin) [inline src/profiles/y_profile_admin_classes.prog.abap:560]: Let's keep it in the database as 'ignore pseudo comments', and 'allow pseudo comments' in the screen. So, we avoid side effects in the already existing profiles.

File: src/profiles/y_profile_admin_classes.prog.abap [hunk 2]
```diff
@@ -818,34 +818,43 @@ CLASS lcl_util IMPLEMENTATION.
   ENDMETHOD.
 
   METHOD init_check_fields_active.
-    DATA obj TYPE REF TO y_check_base.
-
     TRY.
-        IF get_disable_threshold_select( checkid ) EQ abap_true.
+        DATA(obj) = get_check( io_check_id ).
+
+        IF obj->settings-disable_threshold_selection EQ abap_true.
```
Comment (lucasborin) [inline src/profiles/y_profile_admin_classes.prog.abap:824]: Please, use `=` instead of `EQ`

**Comment by lucasborin** [comment]
@eugen-guenther-sap : Review done. Could you please review/implement the changes and create a new pull-request?

File: src/foundation/y_check_base.clas.abap [hunk 3]
```diff
@@ -665,17 +680,15 @@ CLASS y_check_base IMPLEMENTATION.
...
-              p_suppress = settings-pseudo_comment
+              p_suppress = pseudo_comment
               p_param_1 = parameter_01
               p_param_2 = parameter_02
               p_param_3 = parameter_03
               p_param_4 = parameter_04
               p_inclspec = is_include_specific
               p_detail = additional_information
-              p_checksum_1 = checksum
-              p_comments = pseudo_comments ).
```
[the remark this answers is not in the document]
Comment (eugen-guenther-sap) [reply src/foundation/y_check_base.clas.abap:676]: its not used at all

**Pull request merged**